diagrams/02_container.puml

```diff
@@ -17,19 +24,17 @@ System_Boundary(didroom_solution, "DIDroom") {
 }
 
 
-System_Ext(did, "Dyne.org W3C-DID", "Federated 4-levels W3C-DID method with multiple storage mechanisms (DB, Git, blockchain, REDIS, IPFS, file system)")
-System_Ext(relying_party, "Relying party (Zenswarm or 3rd party)", "Microservice verifying verifiable presentations (EUDI-ARF and W3C-VC). Receives a transaction ID in the request from the holder, the transaction ID is generated by the verifier app")
-System_Ext(auth_server, "Authorization server (Zenswarm OR 3rd party software)", "Microservice that verifies the claim of a holder requesting a credential. It issues an `access token` after the holder's request, the access token is sent to the **credential issuer** as an authorization to issue the requested credential")
 
 Rel_R(didroom, db, "Reads from and writes to", "sync, JDBC")
 Rel_R(didroom_backend, db, "Reads only", "sync, JDBC")
 Rel(user, wallet, "Compliant", "EUDI/ARF")
 Rel(verifier, verifier_app, "Compliant", "EUDI/ARF")
 Rel(relying_party, verifier_app, "Compliant", "EUDI/ARF")
 Rel_Neighbor(wallet, relying_party, "Verifiable presentation", "EUDI/ARF")
-Rel(auth_server, issuer , "Send", "access token")
-Rel_R(wallet, auth_server, "Ask", "verifiable credential")
-Rel(issuer, wallet, "Issuance", "verifiable credential and sd-jwt")
+Rel_R(wallet, auth_server, "Ask", "Authorization token")
+Rel(auth_server, wallet , "Send", "Authorization token")
+Rel_R(wallet, issuer, "Send", "Authorization token")
+Rel(issuer, wallet, "Issuance", "Verifiable credential and sd-jwt")
```

Comment:
The purpose of diagram should show the components (and actually containers). You are mixing fluxes. That should go indeed into a sequence diagram.

Reply:
Ok!